protected/humhub/modules/live/controllers/PollController.php

```diff
@@ -53,6 +53,10 @@ public function beforeAction($action)
             return false;
         }
 
+        if (Yii::$app->user->mustChangePassword()) {
```

Comment:
Just for my understanding, this is actually not necessary and also not needed for other similar modules (like TwoFactor Auth), because by the Poll Module variable "isActive" the polling is deactivated anyway?

Reply:
https://github.com/humhub/humhub/pull/4602/files#diff-0dee4426f877787aaeae9be039601f75e47410fba822e0c39ecf8c1cea0e4a7aR35 will deactivate the polling client in the current tab. But the live provider in other browser tabs will still be active. Not sure if this is a problem.

Reply:
Maybe we should add a user setting independent of mustchangepassword if this user setting flag is set, the poll controller returns a 503 https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503.

Reply:
@buddh4 I would leave it at the current solution for now.

A setting is probably not dynamic enough, e.g. with TwoFactor Auth. If it is necessary, however, I would tend to a new event that checks whether the background tasks should be disabled.

Apart from that, had you been able to have a look at the PR? I would like to complete the task otherwise. It looks good to me.

Reply:
Not sure what you mean with "not dynamic enough"? The setting is more or less an abstraction of the current behavior and allows features as "must change password" and "maintainance mode" to inform background services to not deliver content and inform clients as the "poll client" to handle such a state accordingly. Currently the poll client in other tabs will just keep requesting and logging errors. This state configuration won't affect any controller unless a controller like the poll controller manually checks such a state.

Reply:
Because of the "not dynamic" I meant that settings are mostly stored in the database.

How about a simple value in the session? That would be super performant (especially with polling, we should avoid any queries there as much as possible).

Then MustChangePassword,2FA,e.g. could set this session variable (activate limited mode) and then remove it again.
In the case of an error, there would also be no persistent settings in the database.

Reply:
This will only work on a single browser/session basis and not between different devices etc. Currently We need a proper way of informing the frontend about such state changes in the backend and broatcast this state to all tabs/browses/devices which are currently active. Its the same problem with session timeout, user logout, session renew etc. I think we should not discuss this here, but rather open an additional issue/pr for this.

Reply:
@buddh4 Ok I have created an issue here for this. However, it is more of a placeholder, we still need to define more precisely what we need here.
https://github.com/humhub/humhub/issues/4760